src/webgpu/api/operation/rendering/blending.spec.ts

```diff
@@ -502,6 +502,79 @@ g.test('default_blend_constant,initial_blend_constant')
     t.expectOK(result);
   });
 
+g.test('default_blend_constant,setting_blend_constant')
```

Comment:
The default blend constant doesn't have an effect on this test, so maybe rename to just "simple_blend_constant" or something

Reply:
ok, update the prefix of the tests.